arangod/Aql/Optimizer2/PlanNodes/ReturnNode.h

```diff
@@ -26,6 +26,9 @@
 #include "Aql/Optimizer2/PlanNodeTypes/Variable.h"
 #include "Aql/Optimizer2/PlanNodeTypes/Count.h"
 #include "Inspection/VPackInspection.h"
+// TODO: ^-- compiling stops here after I've included ReturnNode into
+// ExecutionNode.cpp with "file not found". Guess, this needs to be fixed in
+// cmake.
```

Comment:
Fails currently with:
```
[ 81%] Building CXX object arangod/Aql/CMakeFiles/arango_aql.dir/IndexHint.cpp.o
In file included from /Users/hkernbach/Git/ArangoDB/devel/arangod/Aql/ExecutionNode.cpp:73:
/Users/hkernbach/Git/ArangoDB/devel/arangod/Aql/Optimizer2/PlanNodes/ReturnNode.h:28:10: fatal error: 'Inspection/VPackInspection.h' file not found
#include "Inspection/VPackInspection.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

we need to fix cmake. up until now, this inspection header file has been used only in our test environment I guess.

Reply:
@markuspf fyi.